mne/beamformer/_lcmv.py

```diff
@@ -423,6 +422,8 @@ def _lcmv_source_power(info, forward, noise_cov, data_cov, reg=0.05,
         info, [info['ch_names'].index(k) for k in ch_names
                if k in info['ch_names']])
 
+    # XXX this could maybe use pca=True to avoid needing to use
+    # _reg_pinv(..., rank=rank) later
```

Comment:
Just a minor comment here that eventually we could make `compute_beamformer` pass on the `pca` argument (we currently don't publicly expose it), meaning that it would return a non-rank-deficient multplier of shape `(rank, 306)` instead of `(306, 306)`. It might simplify things at some point. I've also wanted this parameter before when analyzing data.